keyboards/handwired/orbweaver/keymaps/default/keymap.c

```diff
@@ -0,0 +1,124 @@
+/* Copyright 2023 a_marmot
+*
+* This program is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 2 of the License, or
+* (at your option) any later version.
+*
+* This program is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with this program.  If not, see &lt;http://www.gnu.org/licenses/&gt;.
+*/
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+    /*
+     * ┌───┌───┬───┬───┬───┐
+     * │Esc│ 1 │ 2 │ 3 │ 4 │
+     * ├───┼───┼───┼───┼───┤
+     * │Tab│ q │ w │ e │ r │
+     * ├───┼───┼───┼───┼───┤
+     * │Ctl│ a │ s │ d │ f │
+     * ├───┼───┼───┼───┼───┤
+     * │Shi│ z │ x │ c │ v │
+     * ├───┼───┼───┼───┼───┤
+     * │Alt│ ↑ │ ↓ │ → │ ← │
+     * ├───┼───┼───┴───┴───┤
+     * │Spa│Key│           │
+     * └───┴───┴───────────┘
+     */
+//base layer is a selection menu and bootload access
+    [0] = LAYOUT_keypad(
```

Comment:
```suggestion
    [0] = LAYOUT(
```

Reply:
I originally set this up as a keypad, so this was automatically generated. Does it matter?

Reply:
[PR Checklist](https://docs.qmk.fm/#/pr_checklist):

> - If the keyboard only has a single electrical/switch layout:
>    - use `LAYOUT` as your macro name, unless a community layout already exists

There is no such thing as a `keypad` community layout, so `LAYOUT` with no specific name would be appropriate here (it's specific to this particular board anyway and is not compatible with anything else).